src/app/campaign.resolver.ts

```diff
@@ -35,6 +38,16 @@ export class CampaignResolver implements Resolve<any> {
       );
     }
 
+    if (campaignSlug && fundSlug && campaignSlug !== "campaign") {
```

Comment:
Why do we need to check the following?
`campaignSlug !== "campaign"`

Reply:
My reading of the requriement was we only want to do this redirect when there's a specific metacampaign - e.g.

from https://donate.thebiggive.org.uk/christmas-challenge-2022/xxxxx to https://donate.thebiggive.org.uk/christmas-challenge-2022/ but not from https://donate.thebiggive.org.uk/campaign/abcd to https://donate.thebiggive.org.uk/campaign/